### ISSUE TITLE
[CLI]: Grayscale video

### ISSUE THREAD

**Issue by antonibigata** [wandb/wandb#4619]
### Describe the bug

Hi,

When I try to log grayscale (one channel) videos through the wandb logger on pytorch lightning, I get a weird behavior. The video is getting duplicated 9 times and becomes RGB. This doesn't happen when I manually duplicate the channel dimension to have 3 channels. 

You can see below what happens when I manually make it RGB on the left, and what happens when I just input the video in grayscale on the right (timeXchannelsXheightXwidth).

![wandb_issue](https://user-images.githubusercontent.com/47786928/207368334-90a09357-3392-4c86-84d1-c0d31627a172.png)

### Environment

WandB version: 0.13.4

OS:

Python version:

Versions of relevant libraries:

**Comment by antonibigata** [comment]
Thanks for your quick response!
Here is a temporal workspace I created: https://wandb.ai/animator/temp?workspace=user-bigtoni
and here is the function I use to log the videos:
`def log_videos(self, batch, pred, prefix=""):
        img_cond, _, _, _ = self.read_batch(batch)
        random_idx = torch.randint(0, pred.shape[0], (1,)).item()
        img_cond = unnormalize_to_zero_to_one(img_cond)[random_idx]
        img_pred = rearrange(pred[random_idx], "c t h w -> t c h w")
        repeat_cond = repeat(img_cond, "c h w -> t c h w", t=img_pred.shape[0])
        video = torch.cat([repeat_cond, img_pred], dim=-1).cpu() * 255.0

        if video.shape[1] == 1:
            video = video.repeat(1, 3, 1, 1)

        if self.logger is not None:
            self.logger.experiment.log(
                {
                    f"{prefix}/generated_videos": wandb.Video(
                        video,
                        caption="diffused videos (condition left, generated right)",
                        fps=self.wandb_fps,
                        format="mp4",
                    )
                },
            )`

**Comment by lesliewandb** [comment]
Thank you for all the information on this! I have made a ticket for our engineers and I'll let you know when there are any updates on it
